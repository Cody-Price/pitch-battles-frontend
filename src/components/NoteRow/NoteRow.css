.note-row {
  display: flex;
  justify-content: center;
  margin: auto;
  list-style: none;
  width: 300px;
  padding: 0;
  position: absolute;
  left: 0;
  right: 0;
  bottom: 20px;
}
