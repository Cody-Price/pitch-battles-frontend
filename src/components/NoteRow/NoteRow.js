import React, { Component } from "react";
import Note from "../Note/Note";
import "./NoteRow.css";

class NoteRow extends Component {
  render() {
    const pitches = ["a", "b", "c", "d", "e", "f", "g"];
    const notes = pitches.map(pitch => {
      return <Note pitch={pitch} key={pitch} />;
    });

    return <ul className="note-row">{notes}</ul>;
  }
}

export default NoteRow;
