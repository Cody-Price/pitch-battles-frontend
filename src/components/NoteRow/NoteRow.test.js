import React from "react";
import ReactDOM from "react-dom";
import NoteRow from "./NoteRow";

it("is a test", () => {});
