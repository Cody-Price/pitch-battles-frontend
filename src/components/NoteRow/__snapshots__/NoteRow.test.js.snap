// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoteRow should match the snapshot 1`] = `
<ul
  className="note-row"
>
  <Note
    key="a"
    pitch="a"
  />
  <Note
    key="b"
    pitch="b"
  />
  <Note
    key="c"
    pitch="c"
  />
  <Note
    key="d"
    pitch="d"
  />
  <Note
    key="e"
    pitch="e"
  />
  <Note
    key="f"
    pitch="f"
  />
  <Note
    key="g"
    pitch="g"
  />
</ul>
`;
